src/server/routes/__tests__/catches.integration.spec.js

```diff
@@ -470,4 +470,231 @@ describe('catches.integration', () => {
       expect(result.payload).toBe('')
     })
   })
+
+  describe('PATCH /api/catches/{catchId}', () => {
+    const CONTACT_IDENTIFIER_UPDATE_CATCH = 'contact-identifier-update-catch'
+    beforeEach(
+      async () =>
+        await deleteSubmissionAndRelatedData(CONTACT_IDENTIFIER_UPDATE_CATCH)
+    )
+
+    afterAll(
+      async () =>
+        await deleteSubmissionAndRelatedData(CONTACT_IDENTIFIER_UPDATE_CATCH)
+    )
+
+    const fieldsToTest = [
+      {
+        field: 'dateCaught',
+        value: '2023-05-01T00:00:00+01:00',
+        expected: '2023-05-01'
+      },
+      {
+        field: 'mass',
+        value: { kg: 10, oz: 353, type: 'METRIC' },
+        expected: { kg: 10, oz: 352.739619, type: 'METRIC' }
+      },
+      { field: 'released', value: false, expected: false },
+      { field: 'onlyMonthRecorded', value: true, expected: true },
+      { field: 'noDateRecorded', value: true, expected: true }
+    ]
```

Comment:
don't define this as a separate variable, just put in inside the `each` statement